config/manager/namespace.yaml

```diff
@@ -0,0 +1,10 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  annotations:
+    openshift.io/node-selector: ""
+  labels:
+    controller-tools.k8s.io: "1.0"
+    openshift.io/run-level: "1"
+    openshift.io/cluster-monitoring: "true"
+  name: openshift-cloud-credential-operator
```

Comment:
Duped with https://github.com/openshift/cloud-credential-operator/blob/master/manifests/00_namespace.yaml, can the manager render command use that?

Reply:
the render command has access to generated files out of config/*, but anyway, i thought it best to address this in its own PR #136 as there are subtle things going on here